requirements/base.in

```diff
@@ -2,7 +2,7 @@
 -c constraints.txt
 
 # Django/Framework Packages
-Django>=1.11
+Django>=2.2,<2.3
```

Comment:
What I meant is that we should keep in this file 
`Django>=2.2`

and in the constraints file add the `<2.3` constraint

Reply:
This with the intention to report the minimum version to setup.py